tests/scripts/fetch_manifests.sh

```diff
@@ -28,10 +29,14 @@ manifests=(
   m3datatemplate
 )
 
-for kind in "${manifests[@]}"; do
-  mkdir -p /tmp/manifests/"$kind"
-  for name in $(kubectl get -n metal3 -o name "$kind" || true)
-  do
-    kubectl get -n metal3 -o yaml "$name" | tee /tmp/manifests/"$kind"/"$(basename "$name")".yaml || true
+NAMESPACES="$(kubectl get namespace -o jsonpath='{.items[*].metadata.name}')"
+for NAMESPACE in ${NAMESPACES}
+do
+  for kind in "${manifests[@]}"; do
+    mkdir -p "${DIR_NAME}"/"$kind"
```

Comment:
Let's be consistent with the curly brackets. Maybe also simplify quoting
```suggestion
    mkdir -p "${DIR_NAME}/${kind}"
```